gax-java/gax-grpc/src/main/java/com/google/api/gax/grpc/GrpcCallContext.java

```diff
@@ -602,7 +632,9 @@ public GrpcCallContext withChannel(Channel newChannel) {
         options,
         retrySettings,
         retryableCodes,
-        endpointContext);
+        endpointContext,
+        // Defaults to false again since we cannot tell whether the channel is DirectPath.
+        false);
```

Comment:
Oh shoot, this is a valid public API. I don't see any usages of this in the googleapis org, but can't say for certain this isn't used.

Can we add a warning to the javadocs that passing in a DirectPath channel and specifying CallCreds may lead to issues?

Reply:
> Can we add a warning to the javadocs that passing in a DirectPath channel and specifying CallCreds may lead to issues?

Actually, nvm on this. I think we confirmed this is a non-issue as the servers only use the channelcredential.